plugins/modules/system/xfconf.py

```diff
@@ -10,58 +10,60 @@
 DOCUMENTATION = '''
 module: xfconf
 author:
-    - "Joseph Benden (@jbenden)"
-    - "Alexei Znamensky (@russoz)"
+  - "Joseph Benden (@jbenden)"
+  - "Alexei Znamensky (@russoz)"
 short_description: Edit XFCE4 Configurations
 description:
   - This module allows for the manipulation of Xfce 4 Configuration with the help of
     xfconf-query.  Please see the xfconf-query(1) man pages for more details.
 options:
   channel:
     description:
-    - A Xfconf preference channel is a top-level tree key, inside of the
-      Xfconf repository that corresponds to the location for which all
-      application properties/keys are stored. See man xfconf-query(1).
+      - A Xfconf preference channel is a top-level tree key, inside of the
+        Xfconf repository that corresponds to the location for which all
+        application properties/keys are stored. See man xfconf-query(1).
     required: yes
     type: str
   property:
     description:
-    - A Xfce preference key is an element in the Xfconf repository
-      that corresponds to an application preference. See man xfconf-query(1).
+      - A Xfce preference key is an element in the Xfconf repository
+        that corresponds to an application preference. See man xfconf-query(1).
     required: yes
     type: str
   value:
     description:
-    - Preference properties typically have simple values such as strings,
-      integers, or lists of strings and integers. See man xfconf-query(1).
+      - Preference properties typically have simple values such as strings,
+        integers, or lists of strings and integers. See man xfconf-query(1).
     type: list
     elements: raw
   value_type:
     description:
-    - The type of value being set. This is ignored if the state is "get".
-      For array mode, use a list of types.
+      - The type of value being set.
+      - For array mode: if the list has only one element, use I(force_array=true)
+        to ensure it will be interpreted by the module as a list, otherwise pass
+        a list of types.
```

Comment:
```suggestion
      - "For array mode: if the list has only one element, use I(force_array=true)
         to ensure it will be interpreted by the module as a list, otherwise pass
         a list of types."
```

Reply:
I actually submitted a nother PR to c.g. to further improve the docs on that module. For the 5.0.0 one all you will have to do is remove taht comment:
> This is ignored if the state is "get".